spec/spec_helper_acceptance_local.rb

```diff
@@ -0,0 +1,181 @@
+require 'puppet_litmus'
+require 'singleton'
+
+class Helper
+  include Singleton
+  include PuppetLitmus
+end
+
+SUT_DNS_SERVER = '10.240.1.10'.freeze
+
+RSpec.configure do |c|
+  # Readable test descriptions
+  c.formatter = :documentation
+
+  # Configure all nodes in nodeset
+  c.before :suite do
+    # Make sure the VM is using our internal DNS servers
+    Helper.instance.run_shell("sed -i 's/nameserver.*$/nameserver #{SUT_DNS_SERVER}/' /etc/resolv.conf")
+    Helper.instance.run_shell('puppet module install puppetlabs-inifile')
+    Helper.instance.run_shell('puppet resource package subscription-manager ensure=installed')
+
+    # Defaults to the first satellite found
+    satellite = target_roles('satellite')[0][:name]
+
+    change_target_host(satellite)
+    expand_satellite_disk(satellite) if get_provisioner(satellite) == 'vmpooler' # && satellite['disks']
+    install_satellite(satellite)
+    reset_target_host
+    generate_and_transfer_satellite_cert_from_sat_to_pe(satellite)
+  end
+end
+
+def change_target_host(role)
+  @orig_target_host = ENV['TARGET_HOST']
```

Comment:
I see `change_target_host` being invoked in two `RSpec.before` hooks:

- `:suite`: From [here](https://github.com/puppetlabs/puppetlabs-satellite_pe_tools/pull/129/files#diff-08070fc30be3f9dd014e20b26f80934bR25)
- `:all`: [Calling `satellite_update_setting`](https://github.com/puppetlabs/puppetlabs-satellite_pe_tools/pull/129/files#diff-4bcca2bf146303382123f641fdcb1aa1R11) -> then [here](https://github.com/puppetlabs/puppetlabs-satellite_pe_tools/pull/129/files#diff-08070fc30be3f9dd014e20b26f80934bR170)

Will this mean `@orig_target_host` is what we expect if it's being called twice before the tests execute?

Reply:
change_target_host is only called to set the target to the satellite with reset_target_host swapping it back to the master.

Reply:
`@orig_target_host = ENV['TARGET_HOST']` has been removed